index.html

```diff
@@ -1,237 +1,258 @@
+      </div>
+
+
+      <div class="third_project_mobile">
+        <h3 class="third_project_mtitle">Profesional Art Printing Data</h3>
+        <p class="third_project_mcontent">A daily selection of privately personalized reads; no accounts or sign-ups
+          required. has been the industry's standard</p>
+        <ul class="third_project_mskill_list flex">
+          <li class="third_project_mskills">html</li>
+          <li class="third_project_mskills">Bootstrap</li>
+          <li class="third_project_mskills">Ruby</li>
+        </ul>
+
+        <button class="see_project_button">See Project</button>
+
+      </div>
+
+
+
+
+
+    </div>
+
+  </section>
+
+  <section id="About" class="About_me">
+    <div class="about_me_container">
+      <h2 class="about_me_title">About me.</h2>
+      <p class="about_me_content">Hello I’m a software developer! I can help you build a product , feature or website
+        Look through some of my work and experience! If you like what you see and have a project you need coded, don’t
+        hestiate to contact me.</p>
+
+      <button class="get_resume flex">Get My Resume</button>
+    </div>
+    <div class="about_line">
+      <img src="image folder/Vector 4.png" alt="">
+    </div>
+    <div class="skill_list_container flex">
+      <ul class="skill_list flex">
+        <li class="Language_skill"><img class="border_line" src="image folder/Rectangle55.png" alt="rectangle_icon">
+          Language
+        </li>
+        <div class="Languge_skill_list">
+          <li class="language">Css</li>
+          <li class="language">Ruby</li>
+          <li class="language">Html</li>
+          <li class="language">Javascript</li>
+        </div>
+
+      </ul>
+      <ul class="skill_list flex">
+        <li class="Language_skill2"><img class="border_line" src="image folder/Rectangle56.png" alt="rectangle_icon">
+          Frameworks
+        </li>
+        <div class="Languge_skill_list2">
+          <li class="language">RSpec</li>
+
+          <li class="language">CapyBara</li>
+          <li class="language">Selenium</li>
+          <li class="language">JBootstrap</li>
+          <li class="language">Ruby on Rails</li>
+        </div>
+      </ul>
+      <ul class="skill_list flex">
+        <li class="Language_skill3"><img class="border_line" src="image folder/Rectangle57.png" alt="rectangle_icon">
+          Skills
+        </li>
+        <div class="Languge_skill_list3">
+          <li class="language">Codekit</li>
+          <li class="language">Github</li>
+
+          <li class="language">Gitlab</li>
+          <li class="language">Terminal</li>
+          <li class="language">Codepen</li>
+        </div>
+      </ul>
+
+
+    </div>
+  </section>
+  <section id="form_container_link Contact" class="form_section">
+    <div class="form_container">
+      <h3 class="form_text">I'm always interested in hearing about new projects, so if you'd like to chat please get in
+        touch..</h2>
```

Comment:
- [ ] Please be extra careful with opening and closing tags, here you're opening with a `<h3>` tag and closing with a `<h2>`.